.github/workflows/test.yml

```diff
@@ -1,13 +1,13 @@
 on: [push, pull_request]
 name: Test
 jobs:
-  test:
+  version:
     name: Test
     permissions:
       contents: read
     strategy:
       matrix:
-        go-version: ['1.14.15', '1.15.15', '1.16.15', '1.17.13', '1.22.3']
+        go-version: ['oldstable', 'stable']
```

Comment:
My gut feeling is that this should match go.mod and then run to stable.

I didn't know it supported `stable` even!

Reply:
Stable is pretty cool though.

Reply:
Unfortunately because one uses `go-version: <version>` and the other uses `go-version-file: <path>`, they can't live in the same matrix.